admin_guide/scheduler.adoc

```diff
@@ -367,3 +367,205 @@ Since the scheduler is a plug-in, it can be replaced in favor of an alternate
 implementation. The scheduler code has a clean separation that watches new pods
 as they get created and identifies the most suitable node to host them. It then
 creates bindings (pod to node bindings) for the pods using the master API.
+
+
+[[controlling-pod-placement]]
+== Controlling Pod Placement
+
+As a cluster administrator, you can set a policy to prevent application
+developers with certain roles from targeting specific nodes when scheduling
+pods. 
+
+[IMPORTANT]
+====
+This process involves the *pods/binding* permission
+link:../admin_guide/manage_authorization_policy.html[role], which is needed to
+target particular nodes. The constraint on the use of the `nodeSelector` field
+of a pod configuration is based on the *pods/binding* permission and the
+`nodeSelectorLabelBlacklist` configuration option.
+====
+
+The `nodeSelectorLabelBlacklist` field of a master configuration file gives you
+control over the labels that certain roles can specify in a pod configuration's
+`nodeSelector` field. Users, service accounts, and groups that have the
+*pods/binding* permission can specify any node selector. Those without the
+*pods/binding* permission are prohibited from setting a `nodeSelector` for any
+label that appears in `nodeSelectorLabelBlacklist`.
+
+As a hypothetical example, an OpenShift cluster might consist of five data
+centers spread across two regions. In the U.S., "us-east", "us-central", and
+"us-west"; and in the Asia-Pacific region (APAC), "apac-east" and "apac-west".
+Each node in each geographical region
+link:../admin_guide/manage_nodes.html#updating-labels-on-nodes[should be labeled
+accordingly]. For example, `region: us-east`.
+
+As a cluster administrator, you can create an infrastructure where application
+developers should be deploying pods only onto the nodes closest to their
+geographical location. You can
+link:../admin_guide/managing_projects.html#using-node-selectors[create a node
+selector], grouping the U.S. data centers into `superregion: us` and the APAC
+data centers into `superregion: apac`.
+
+To maintain an even loading of resources per data center, you can add the
+desired `region` label to the `nodeSelectorLabelBlacklist` section of a master
+configuration. Then, whenever a developer located in the U.S. creates a pod, it
+is deployed onto a node in one of the regions with the `superregion: us` label.
+If the developer tries to target a specific region for their pod (for example,
+`region: us-east`), they will receive an error. If they try again, without the
+node selector on their pod, it can still be deployed onto the region they tried
+to target, because `superregion: us` is set as the project-level node selector,
+and nodes labeled `region: us-east` are also labeled `superregion: us`.
+
+[[constraining-pod-placement-labels]]
+=== Constraining Pod Placement Using Node Labels
```

Comment:
This part should be "Constraining Pod Placement Using Node Name".  it's not related to node label. if you enable this controller, no matter what's the nodeName is it will rejected to create.